### PR TITLE
fix(GRO-1645): add padding to results grid 

### PR DESCRIPTION
**Pull request by laurabeth**
This PR resolves [GRO-1645] & [GRO-1609] 

### Description

- Adds padding between the Top Auction Lots header and artwork grid
- Updates copy throughout the art quiz
- Adds logic in art quiz results to switch header text based on the tab you're on, and a test to verify that behavior

Simulator Screenshot - iPhone 14 Pro Max - 2023-05-16 at 15 38 11

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[GRO-1645]: https://artsyproduct.atlassian.net/browse/GRO-1645?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[GRO-1609]: https://artsyproduct.atlassian.net/browse/GRO-1609?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Scenes/ArtQuiz/ArtQuizResults/ArtQuizResults.tests.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      "Works for You",
+      "Explore Your Quiz Results",
+      "We think you’ll enjoy these recommendations based on your likes. To tailor Artsy to your art tastes, follow artists and save works you love.",
+    ],
+    [
+      "Artists for You",
+      "Explore Your Quiz Results",
+      "We think you’ll enjoy these recommendations based on your likes. To tailor Artsy to your art tastes, follow artists and save works you love.",
+    ],
+  ])("should display expected header for %s tab", (selected, title, subtitle) => {
```
Comment (gkartalis) [inline src/app/Scenes/ArtQuiz/ArtQuizResults/ArtQuizResults.tests.tsx:40]: **praise:** super cool 👏

File: src/app/Scenes/ArtQuiz/ArtQuizResults/ArtQuizResults.tests.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    const { debug } = renderWithRelay()
+
+    const tab = screen.getByText(selected)
+
+    fireEvent.press(tab)
+    expect(screen.getByText(title)).toBeTruthy()
+    expect(screen.getByText(subtitle)).toBeTruthy()
+
+    debug()
```
Comment (gkartalis) [inline src/app/Scenes/ArtQuiz/ArtQuizResults/ArtQuizResults.tests.tsx:49]: **non-blocking:** we could get rid of the debug to reduce noise on the ci but leaving this up to you 🤘

File: src/app/Scenes/Onboarding/OnboardingQuiz/OnboardingMarketingCollection.tsx [hunk 3]
```diff
@@ -46,6 +46,7 @@ const OnboardingMarketingCollection: React.FC<OnboardingMarketingCollectionProps
           description={description}
           marketingCollection={marketingCollection!}
         />
+        <Spacer y={2} />
```
Comment (gkartalis) [inline src/app/Scenes/Onboarding/OnboardingQuiz/OnboardingMarketingCollection.tsx:49]: 👏

**Review by gkartalis — DISMISSED** [review]
🌟 thanks!

File: src/app/Scenes/ArtQuiz/ArtQuizResults/ArtQuizResults.tests.tsx (hunk 2 above)
Comment (laurabeth) [reply src/app/Scenes/ArtQuiz/ArtQuizResults/ArtQuizResults.tests.tsx:49]: 🤡  I always forget one.

**Pull request merged**